packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts

```diff
@@ -0,0 +1,111 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { BalanceTransferPayload, TransferDomainType } from '../../../src/category/account'
+import { RpcApiError } from '@defichain/jellyfish-api-core/dist/index'
+import BigNumber from 'bignumber.js'
+
+describe('TransferDomain', () => {
+  let dfiAddress: string, ethAddress: string
+  const amountToTransfer = 5
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForWalletCoinbaseMaturity()
+    await client.masternode.setGov({
+      ATTRIBUTES: {
+        'v0/params/feature/evm': 'true'
+      }
+    })
+    await container.generate(1)
+    dfiAddress = await container.call('getnewaddress')
+    // transfer utxostoaccount
+    await container.call('utxostoaccount', [{ [dfiAddress]: '100@0' }])
+    await container.generate(1)
+    ethAddress = await container.getNewAddress('', 'eth')
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should fail Transfer Balance from DFC to EVM if input and output value is different', async () => {
```

Comment:
For consistency on naming
```suggestion
  it('should fail Transfer Domain from DFC to EVM if from and to value is different', async () => {
```

Reply:
same with all other `it`s